spec/features/users/index_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe 'users#index' do
+  before :each do
+    @user1 = User.create!(name: 'Alice', bio: 'Maths teacher',
+                          photo: 'https://source.unsplash.com/user/c_v_r/1900x800', post_counter: 0)
+    @user2 = User.create!(name: 'Bob', bio: 'Computer science lecturer',
+                          photo: 'https://source.unsplash.com/user/c_v_r/100x100', post_counter: 0)
+
+    Post.create!(author: @user1, title: 'Hello there', text: 'This is my first post')
+    Post.create!(author: @user2, title: 'Hello there', text: 'This is my first Blog post')
+    Post.create!(author: @user1, title: 'Hello mate', text: 'This is my second blog post')
+    Post.create!(author: @user2, title: 'Hello there mate', text: 'This is my second blog post')
+
+    visit users_path
+  end
+
+  it 'should display all users' do
+    expect(page).to have_content("Username: #{@user1.name}")
+    expect(page).to have_content("Username: #{@user2.name}")
+  end
+
+  it 'should see user profile picture' do
+    if @user1.photo
+      expect(page).to have_css("img[src*='https://source.unsplash.com/user/c_v_r/1900x800']")
+    else
+      expect(page).to have_content(user.photo)
+    end
+  end
+
+  it 'should show links to view post for a single user' do
+    expect(page).to have_link(@user1.name, href: user_path(id: @user1.id))
+    expect(page).to have_link(@user2.name, href: user_path(id: @user2.id))
+  end
+
+  it 'should see the number of posts by each user' do
+    expect(page).to have_content('2')
+  end
```

Comment:
- According to the project requirements, please test that the number of posts written by each user can be seen. From what you have done, you have only checked if the page contains the number `2`. Please fix this 🙏